packages/suite/src/views/recovery/index.tsx

```diff
@@ -208,41 +236,18 @@ const Recovery = ({ modal, closeModalApp }: InjectedModalApplicationProps) => {
                 )}
 
                 {recovery.status === 'initial' && (
-                    <>
-                        <CheckItem
-                            data-test="@recovery/user-understands-checkbox"
-                            title={<Translation id="TR_DRY_RUN_CHECK_ITEM_TITLE" />}
-                            description={<Translation id="TR_DRY_RUN_CHECK_ITEM_DESCRIPTION" />}
-                            isChecked={understood}
-                            link={
-                                <TrezorLink
-                                    icon="EXTERNAL_LINK"
-                                    size="tiny"
-                                    href={URLS.DRY_RUN_URL}
-                                >
-                                    <Translation id="TR_LEARN_MORE" />
-                                </TrezorLink>
-                            }
-                            onClick={() => setUnderstood(!understood)}
-                        />
-
-                        <Buttons>
-                            <StyledButton
-                                onClick={() =>
-                                    model === 1
-                                        ? actions.setStatus('select-word-count')
-                                        : actions.checkSeed()
-                                }
-                                isDisabled={!understood || isLocked()}
-                                data-test="@recovery/start-button"
-                            >
-                                <Translation id="TR_START" />
-                            </StyledButton>
-                            <CloseButton onClick={() => closeModalApp()}>
-                                <Translation id="TR_CANCEL" />
-                            </CloseButton>
-                        </Buttons>
-                    </>
+                    <CheckItem
+                        data-test="@recovery/user-understands-checkbox"
+                        title={<Translation id="TR_DRY_RUN_CHECK_ITEM_TITLE" />}
+                        description={<Translation id="TR_DRY_RUN_CHECK_ITEM_DESCRIPTION" />}
+                        isChecked={understood}
+                        link={
+                            <TrezorLink icon="EXTERNAL_LINK" size="tiny" href={URLS.DRY_RUN_URL}>
+                                <Translation id="TR_LEARN_MORE" />
+                            </TrezorLink>
+                        }
+                        onClick={() => setUnderstood(!understood)}
+                    />
                 )}
 
                 {recovery.status === 'select-word-count' && (
```

Comment:
Perhaps put it in a `switch`? Would make the render part of a component more readable.

Reply:
Should have mentioned that I meant a switch should not be in the render itself, rather smth like a `getStep()` func which would be called in the render. Up to you :)